### PR TITLE
Keyword updates. addresses issue #281 issue #248

### PR DESCRIPTION
**Pull request by kelle**
lots of tweaks on many tutorials, adding keywords, expanding learning goals, clarifying language, fixing links, hiding warnings, etc.

Also big expansion of tutorial keywords on the html. There may be some missing divs.

### CODE REVIEW

File: tutorials/notebooks/FITS-cubes/FITS-cubes.ipynb [hunk 1]
```diff
@@ -870,21 +870,21 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
    "language": "python",
-   "name": "python3"
+   "name": "python2"
```
Comment (eblur) [inline tutorials/notebooks/FITS-cubes/FITS-cubes.ipynb]:
```suggestion
   "name": "python3"
```

File: tutorials/notebooks/FITS-cubes/FITS-cubes.ipynb [hunk 2]
```diff
@@ -870,21 +870,21 @@
...
   },
   "language_info": {
    "codemirror_mode": {
     "name": "ipython",
-    "version": 3
+    "version": 2
```
Comment (eblur) [inline tutorials/notebooks/FITS-cubes/FITS-cubes.ipynb]:
```suggestion
    "version": 3
```

File: tutorials/notebooks/FITS-cubes/FITS-cubes.ipynb [hunk 3]
```diff
@@ -870,21 +870,21 @@
...
    },
    "file_extension": ".py",
    "mimetype": "text/x-python",
    "name": "python",
    "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.7.2"
+   "pygments_lexer": "ipython2",
```
Comment (eblur) [inline tutorials/notebooks/FITS-cubes/FITS-cubes.ipynb]:
```suggestion
   "pygments_lexer": "ipython3",
```

File: tutorials/notebooks/FITS-cubes/FITS-cubes.ipynb [hunk 4]
```diff
@@ -870,21 +870,21 @@
...
+   "version": "2.7.13"
```
Comment (eblur) [inline tutorials/notebooks/FITS-cubes/FITS-cubes.ipynb]:
```suggestion
   "version": "3.7.2"
```

File: tutorials/notebooks/FITS-tables/FITS-tables.ipynb [hunk 5]
```diff
@@ -353,21 +383,21 @@
    "published": true
   },
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
    "language": "python",
-   "name": "python3"
+   "name": "python2"
```
Comment (eblur) [inline tutorials/notebooks/FITS-tables/FITS-tables.ipynb]:
```suggestion
   "name": "python3"
```

File: tutorials/notebooks/FITS-tables/FITS-tables.ipynb [hunk 6]
```diff
@@ -353,21 +383,21 @@
...
   },
   "language_info": {
    "codemirror_mode": {
     "name": "ipython",
-    "version": 3
+    "version": 2
```
Comment (eblur) [inline tutorials/notebooks/FITS-tables/FITS-tables.ipynb]:
```suggestion
    "version": 3
```

File: tutorials/notebooks/FITS-tables/FITS-tables.ipynb [hunk 7]
```diff
@@ -353,21 +383,21 @@
...
    },
    "file_extension": ".py",
    "mimetype": "text/x-python",
    "name": "python",
    "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.7.3"
+   "pygments_lexer": "ipython2",
```
Comment (eblur) [inline tutorials/notebooks/FITS-tables/FITS-tables.ipynb]:
```suggestion
   "pygments_lexer": "ipython3",
```

File: tutorials/notebooks/FITS-tables/FITS-tables.ipynb [hunk 8]
```diff
@@ -353,21 +383,21 @@
...
+   "version": "2.7.13"
```
Comment (eblur) [inline tutorials/notebooks/FITS-tables/FITS-tables.ipynb]:
```suggestion
   "version": "3.7.3"
```

File: tutorials/notebooks/UVES/UVES.ipynb [hunk 9]
```diff
@@ -1320,21 +1349,21 @@
    "published": true
   },
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
    "language": "python",
-   "name": "python3"
+   "name": "python2"
```
Comment (eblur) [inline tutorials/notebooks/UVES/UVES.ipynb]:
```suggestion
   "name": "python3"
```

File: tutorials/notebooks/UVES/UVES.ipynb [hunk 10]
```diff
@@ -1320,21 +1349,21 @@
...
   },
   "language_info": {
    "codemirror_mode": {
     "name": "ipython",
-    "version": 3
+    "version": 2
```
Comment (eblur) [inline tutorials/notebooks/UVES/UVES.ipynb]:
```suggestion
    "version": 3
```

File: tutorials/notebooks/UVES/UVES.ipynb [hunk 11]
```diff
@@ -1320,21 +1349,21 @@
...
    },
    "file_extension": ".py",
    "mimetype": "text/x-python",
    "name": "python",
    "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.6.5"
+   "pygments_lexer": "ipython2",
```
Comment (eblur) [inline tutorials/notebooks/UVES/UVES.ipynb]:
```suggestion
   "pygments_lexer": "ipython3",
```

File: tutorials/notebooks/UVES/UVES.ipynb [hunk 12]
```diff
@@ -1320,21 +1349,21 @@
...
+   "version": "2.7.13"
```
Comment (eblur) [inline tutorials/notebooks/UVES/UVES.ipynb]:
```suggestion
   "version": "3.6.5"
```

**Comment by kelle** [comment]
All notebooks have been re-run with Python3 kernel. but now tests are failing because of something with changing with servers from which data/info is downloaded. I started a slack convo (astropy #general) but a solution has not yet emerged.

**Comment by kelle** [comment]
all right, this is working now with three tutorials removed! Yay!  I'm not sure what to do next...should I close this PR and start a new one just with the 15 tutorials which work? @adrn ?

**Pull request closed without merging**